tracim/tracim/lib/content.py

```diff
@@ -860,6 +860,47 @@ def move(self, item: Content,
 
         item.revision_type = ActionDescription.MOVE
 
+    def copy(
+        self,
+        item: Content,
+        new_parent: Content=None,
+        new_label: str=None,
+        do_save: bool=True,
+    ) -> None:
+        if not new_parent and not new_label:
+            # TODO - G.M - 08-03-2018 - Use something else than value error
+            raise ValueError("You can't copy file into itself")
```

Comment:
If `new_parent` and `new_label` are filled but with same parent and label than source the copy should fail too.